integration-tests/tests.py

```diff
@@ -21,18 +21,47 @@ def run_tests(behaviour, args):
             if exists[0] != exists[1]:
                 sys.exit(f"{exists[0]} of {exists[1]} keys, {behaviour} test failed, see output")
             else:
-                print(f"{exists[0]} of {exists[1]} keys")   
+                print(f"{exists[0]} of {exists[1]} keys")
         else:
             sys.exit('Unknown behaviour.')     
     except subprocess.CalledProcessError as e:
         sys.exit(str(e.stderr))
     except Exception as e:
         sys.exit(str(e))
 
+def run_doubled_exist_test(run_args, expected_exist_keys):
+    try:
+        run_args['-c'] = expected_exist_keys * 2 + 1
+        run_args['-s'] = 5001
+        run_args['-t'] = 1
+        args = make_args(run_args)
```

Comment:
`make_args` declared blow. Due to this the function probably is not available here